data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleBuilder.java

```diff
@@ -211,11 +214,29 @@ private static Filter getFilter(DataPlaneContract.DialectedFilter filter) {
     };
   }
 
+  private WebClientOptions createWebClientOptionsFromCACerts(String CACerts) {
+    if (!CACerts.isEmpty()) {
```

Comment:
Might `CACerts` ever be `null`?

Reply:
I'm not sure? When I added a !=null check IntelliJ said that it would always evaluate to true so I guessed it wouldn't but I am not very familiar with this part of the codebase.

Reply:
Sounds good, just wanted to make sure we've considered that case

Reply:
Just in case one of them can be null (I only checked for one of the CA certs variables before I settled on the approach of having a private method for this), I added a null check